Patterns/French/French-DateTime.yaml

```diff
@@ -479,12 +479,12 @@ AgoPrefixRegex: !simpleRegex
 LaterRegex: !simpleRegex
   def: \b(plus\s+tard|à\s+partir\s+(de\s+(maintenant|demain)|d'aujourd'hui)|après\s+(aujourd'hui|demain))\b
 AgoRegex: !nestedRegex
-  def: \b((depuis|il\s+y\s*a)(\s+{AroundRegex})?|auparavant|avant\s+(?<day>hier|aujourd'hui))\b
+  def: \b((il\s+y\s*a)(\s+{AroundRegex})?|auparavant|avant\s+(?<day>hier|aujourd'hui))\b
   references: [ AroundRegex ]
 BeforeAfterRegex: !simpleRegex
   def: ^\b$
 InConnectorRegex: !simpleRegex
-  def: \b(dans|en|sur)\b
```

Comment:
Why did we remove sur?

Reply:
> Why did we remove sur?

InConnectorRegex is used to extract the connector "in" in cases like "In 3 days/weeks/months/years" which will be recognized as date. 
"sur 3 ans" means "over 3 year" and it expected to be a duration, "sur" means "over" and not in. If we keep the sur here, it will be recognized as date.